src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable */
+import Calculator from "./components/Calculator";
+import "./App.css";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator></Calculator>
```

Comment:
- [ ] Please make the App a class based component aswell as was required in the project guidelines. 
**_Use class based components (All components should be class-based)._**